src/App.js

```diff
@@ -1,7 +1,290 @@
+import React from "react";
+import styled from "styled-components";
+
+const SidebarContainer = styled.div`
+  width: 250px;
+  height: 100%;
+  padding: 16px;
+  background-color: white;
+  position: absolute;
+  top: 0;
+  right: 0;
+`;
+
+
+
+const UserInfo = styled.div`
+  display: flex;
+  align-items: center;
+  margin-bottom: 20px;
+`;
+
+const ProfileImage = styled.img`
+  width: 50px;
+  height: 50px;
+  border-radius: 50%;
+  margin-right: 10px;
+`;
+
+const UserName = styled.div`
+  font-weight: bold;
+`;
+
+const Suggestions = styled.div`
+  margin-top: 20px;
+`;
+
+const SuggestionsList = styled.ul`
+  list-style-type: none;
+  padding: 0;
+`;
+
+const SuggestionItem = styled.li`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: 10px;
+`;
+
+const SuggestedProfileImage = styled.img`
+  width: 30px;
+  height: 30px;
+  border-radius: 50%;
+`;
+
+const FollowButton = styled.button`
+  border: none;
+  background-color: #0095f6;
+  color: white;
+  padding: 5px 10px;
+  border-radius: 5px;
+  font-weight: bold;
+  cursor: pointer;
+`;
+
+function InstagramSidebar() {
+  return (
+    <SidebarContainer>
+      <UserInfo>
+        <ProfileImage
+          className="profileImage"
+          src="https://via.placeholder.com/50"
+          alt="프로필 이미지"
+        />
+        <div>
+          <UserName>huise0ng</UserName>
+          <div className="username">희성٭</div>
+        </div>
+      </UserInfo>
+      <Suggestions>
+        <h5>회원님을 위한 추천</h5>
+        <SuggestionsList>
+          {['cys_070124', 'cys_070124', 'cys'].map((friend, index) => (
+            <SuggestionItem key={index}>
+              <SuggestedProfileImage
+                className="suggestedProfileImage"
+                src={`https://via.placeholder.com/30`}
```

Comment:
그냥 string이면 템플릿 리터럴 사용은 지양하는게 좋습니다
```suggestion
                src="https://via.placeholder.com/30"
```